Makefile

```diff
@@ -118,6 +120,9 @@ it38: check-venv python-caches-clean tox-env-clean
 it39: check-venv python-caches-clean tox-env-clean
 	. $(VENV_ACTIVATE_FILE); tox -e py39
 
+it310: check-venv python-caches-clean tox-env-clean
```

Comment:
This target (and `it39`) is missing from the phony targets. Can you please add both there? We should probably declare them as phony together with the task definition rather than at the end of the build file (could be a follow-up).